lib/groonga-query-log/command/run-regression-test.rb

```diff
@@ -187,6 +188,14 @@ def create_option_parser
                   "specifying this option multiple times") do |accessor|
           @nullable_reference_number_accessors << accessor
         end
+        parser.on("--[no-]rewrite_regular_expression",
```

Comment:
Could you use `-` as word separator for consistency?

Reply:
This name is too generic.
We don't rewrite all regular expression, right?

`--rewrite-not-or-regular-expression` or something will be better.